website/src/views/Signin/Signin.jsx

```diff
@@ -85,6 +121,28 @@ export default class Signin extends PureComponent {
             <Button size="large" block style={{ marginTop: 10 }} href="/signup">
               <p style={{ margin: 0, display: 'inline' }}>Don&#39;t have an account? Sign Up.</p>
             </Button>
+            <Button size="large" block style={{ marginTop: 10 }} onClick={this.forgotPassword}>
+              <p style={{ margin: 0, display: 'inline' }}>Forgot Password?</p>
+            </Button>
+            <Modal
+              title="Forgot password"
+              className="forgot-password-modal"
+              visible={visible}
+              onOk={this.handleOk}
```

Comment:
Some styling changes to make sure this modal has the same styling as the `JoinClass` modal.

Add `centered` tag here, so that the modal is centered in screen, something like:
```js
<Modal
    centered
    title="Forgot Password" // please capitalize the 'P'
    ...

```

In `Signin.css` add the following:
```css
.forgot-password-modal .ant-modal-content,
.ant-modal-header {
  border-radius: 15px !important;
}
```